src/hooks/index.js

```diff
@@ -9,4 +11,19 @@ const useChangeHandler = setState => {
   return onChangeHandler;
 };
 
-export { useChangeHandler };
+const useFormatMessage = (
+  id,
+  values = {},
+  defaultMessage = '',
+  description = ''
+) => {
+  const intl = useIntl();
+  return intl.formatMessage({ id, defaultMessage, description }, values);
+};
+
+const useFormatDate = (value, weekday, year, month, day) => {
```

Comment:
Weekday, year, month, day can be passed as an object and this object should be optional.